### ISSUE TITLE
transfer and rename this repository

### ISSUE THREAD

**Issue by jrhillae** [inbo/riparias-prep#47]
@SanderDevisscher: 

Is all code within this repository used for D1? Otherwise it can be separated in two distinct repositories?

Damiano and I suggest to either transfer the code of the Rshiny app to a new repository and/or rename this repository before the deliverable is written of D1. 

The deliverable of D1 needs to refer to the source code and documentation of the app, which for clarity should be collected in a repository with a clear name (i.e. monitoring management) , only containing code related to the app, within organisation LIFE RIPARIAS.

**Comment by SanderDevisscher** [comment]
> @SanderDevisscher:
> 
> Is all code within this repository used for D1? Otherwise it can be separated in two distinct repositories?

NO, it isn't I think. D1 is mainly the app while some scripts data also concern the crayfish and plants climate matching & a baseline maps shinyapp for zwalm based on the one I made for RIPARIAS which you greatly enhanced.
> 
> Damiano and I suggest to either transfer the code of the Rshiny app to a new repository and/or rename this repository before the deliverable is written of D1.

Good idea to keep the code seperate & clean
> 
> The deliverable of D1 needs to refer to the source code and documentation of the app, which for clarity should be collected in a repository with a clear name (i.e. monitoring management) , only containing code related to the app, within organisation LIFE RIPARIAS.

**Comment by timadriaens** [comment]
I would suggest moving everything related to RIPARIAS to the public riparias repo i.e. both the climate matching for the alert list drafting as well as the code for D1. Does that seem feasible?

**Comment by SanderDevisscher** [comment]
This repo is public too

Op za 16 apr. 2022 15:11 schreef Tim Adriaens ***@***.***>:

**Comment by SanderDevisscher** [comment]
Has this been done ?

**Comment by damianooldoni** [comment]
No, not yet. I just wanted to be sure you agee first. And I didn't want to do it just in the middle of some merging or PR as you all seem very active in this repo.

Let me know which period is the best one to transfer this repo.
Can we transfer climate matching too at the same moment or better waiting for it?

**Comment by SanderDevisscher** [comment]
> No, not yet. I just wanted to be sure you agee first. And I didn't want to do it just in the middle of some merging or PR as you all seem very active in this repo.

ok, we'll let you know.

> Let me know which period is the best one to transfer this repo. Can we transfer climate matching too at the same moment or better waiting for it?

I think its best to keep the climate matching separate from the shinyapp code. The question then would be, is it necessary to move this part of the repo because this repo is also public.

**Comment by damianooldoni** [comment]
I know everything is public. It's just a question of putting RIPARIAS things within the RIPARIAS GitHub organization. Up to you guys to decide what officially falls under LIFE RIPARIAS and what not. And also how feasible is to move stuff without breaking anything. IMHO climate matching is part of LIFE RIPARIAS. If this repo will not be active anymore I would not move it. 

I agree to not mix the RShiny code with climate matching in the same repo.

**Comment by jrhillae** [comment]
@SanderDevisscher : This has not yet been done.
